Fuga.php

```diff
@@ -8,10 +8,11 @@
 class Fuga {
 
     public function index() {
-        return true;
+        $i = 0;
+        if($i == 1)
+            return true;
+        else
```

Comment:
__[code_sniffer]__
* Expected 1 space after ELSE keyword; newline found
* Inline control structures are not allowed